### ISSUE TITLE
A question about reprocuctive results on FEDformer

### ISSUE THREAD

**Issue by maxwell-five** [cure-lab/LTSF-Linear#37]
Hello,In the process of reproducing your paper, I encountered a little problem about FEDformer.
I did what you told at readme.md
>torch1.11) [xxx@master FEDformer]sh scripts/LongForecasting.sh

And some error happened
```shell
RuntimeError: Given groups=1, weight of size [512, 7, 3], expected input[32, 1, 98] to have 7 channels, but got 1 channels instead
```
Have you meet this situation before?

**Comment by mixiancmx** [comment]
No. But this error might be because of the value of args.enc_in, args.dec_in and args.c_out are not equal to the number of channels in the dataset. To debug, you can check the value of these arguments

**Comment by maxwell-five** [comment]
Thanks for your useful suggestion and impressive work .
